gwpy/cli/tests/test_histogram.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) Duncan Macleod (2018-2021)
+#
+# This file is part of GWpy.
+#
+# GWpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# GWpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with GWpy.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Unit tests for :mod:`gwpy.cli.histogram`
+"""
+
+from ... import cli
+from .base import (_TestTimeDomainProduct, update_namespace)
+
+
+class TestCliTimeSeries(_TestTimeDomainProduct):
+    TEST_CLASS = cli.TimeSeries
```

Comment:
Looks like CircleCI is seeing a failure related to class definitions:

```
self = <gwpy.cli.tests.test_histogram.TestCliTimeSeries object at 0x7fd8860f6400>
prod = <gwpy.cli.timeseries.TimeSeries object at 0x7fd885b9d340>

    def test_action(self, prod):
        if self.ACTION is None:
            raise NotImplementedError
>       assert prod.action is self.ACTION
E       AssertionError

/usr/local/lib/python3.8/site-packages/gwpy/cli/tests/base.py:170: AssertionError
```

Is there a simple fix for this?

Reply:
I copied that from the time series test, maybe I missed something. Checking

Reply:
Found that I believe I was not consistent trying to test the histogram product with the cli.TimeSeries class.

Reply:
I made enough mods to get the tests to pass